scripts/ai2_internal/resume_daemon.py

```diff
@@ -0,0 +1,219 @@
+#! /usr/bin/env python
+
+# Tool to automatically resume preemptible beaker experiments.
+
+from sqlite3 import Connection
+from enum import Enum
+import argparse
+import json
+import logging
+import os
+import sqlite3
+import subprocess
+from logging.handlers import RotatingFileHandler
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+logger.setLevel(logging.DEBUG)
+handler = RotatingFileHandler(
+        f"{os.environ['HOME']}/.allennlp/resume.log",
+        maxBytes=1024*1024,
+        backupCount=10)
+logger.addHandler(handler)
+
+BEAKER_QUERY_INTERVAL_SECONDS = 1
```

Comment:
Unused?

Reply:
Thanks for catching this. I've added some sleeps to smooth the load to beaker. I don't we'll ever have the number of users for this to really matter, but just as a matter of good practice I've added this in.